src/routes/BookmarkedMovies.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ Create a new page
+ Import the necessary components
+ Follow the design on Figma
+ Display bookmarked movies (use dummy data for now).
+ */
+
+import { NavBar } from "../components/nav-bar";
+import { PageButton } from "../components/page-button";
+const BookmarkedMovies = () => {
+  return (
+    <div>
+      <div className="grid grid-cols-2 gap-5">
+        <div>
+          <img
+            src="/public/img/im-westen-nichts-neues.jpg"
```

Comment:
Files in the public directory are served at the root path.
Instead of /public/img/im-westen-nichts-neues.jpg, use /img/im-westen-nichts-neues.jpg.
Change this for all images.